src/main/game/Line.java

```diff
@@ -0,0 +1,33 @@
+package main.game;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Arrays.asList;
+
+public class Line {
+    private List<String> spaces;
+
+    public Line(String spaceOne, String spaceTwo, String spaceThree) {
+        spaces = asList(spaceOne, spaceTwo, spaceThree);
+    }
+
+    public boolean hasWinner(String mark) {
+        int markCounter = 0;
+
+        for (String space : spaces) {
+            if (space.equals(mark)) {
+                markCounter++;
```

Comment:
Really small thing - here you increment using the `++` and in board.isMoveAvailable() you increment by doing `+= 1`

It would be nice if these were consistent (this is real nit-picking!)